api/graphql/schema.graphql

```diff
@@ -521,6 +523,7 @@ type Team {
   description: LocalizedString
   departments: [Department] @belongsToMany
   contactEmail: Email @rename(attribute: "contact_email")
+  pools: [Pool] @hasMany
```

Comment:
This is a potential security leak, since a user who has permission to view a Team may not have permissions to view Pools. (Remember, we generally use Pool as a highly-restricted type, with PoolAdvertisement as the public version.) 
This may likely get changed during the course of the Roles and Teams epic, but for now, the quickest solution would be to slap `@can(ability: "viewAny")` on Team.pools field, as well as on any `team`/`teams` fields which appear in the schema.

Reply:
okay, slapped the directive on in a couple places https://github.com/GCTC-NTGC/gc-digital-talent/pull/5561/commits/6fca42cdb5b89926cdc5af6f06768bfe48fb346c